manifests/examples/delete-br1-linux-bridge.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: nmstate.io/v1
+kind: NodeNetworkState
+metadata:
+  name: node01
+spec:
+  managed: true
+  nodeName: node01
+  desiredState:
+    interfaces:
+      - name: br1
+        type: linux-bridge
+        state: absent
```

Comment:
do we need to do this? I though that just removing br1 from the list would work (as long as it was previously created by us). If we configured br1 and br2, this would remove both of them (AFAIK). Maybe we can keep one manifest that would do the cleanup.

Reply:
Have just test it and it's not working, only explicit state specification as state at nmstate examples 
https://github.com/nmstate/nmstate/blob/master/examples/ovsbridge_delete.yml

Reply:
hello @EdDev @tyll, could you please shed some light on this one? With nmstatectl, if I apply non-empty state.yaml and then apply and empty one, does the empty one remove everything configured in the first non-empty state?

Reply:
If you want to remove an interface (also true for route, dns, etc), you need to explicitly mention it. Interfaces that are not mentioned are ignored, allowing partial editation without requiring to specify the whole state.
I guess we can support a fill-state definition, but it will not scale IMO. Clients are interested in a partial scope of the configuration, requiring them to manage all the configuration is a burden.

Reply:
Note that there are exceptions to this, specifically to the IP addresses and Bridge/Bond ports. There, the user needs to explicitly give the whole list.
It is introducing an inconsistency in favour of practical usage. Obviously, we may find out we need to adjust to the practical usage if needed.

Reply:
I see its value in nmstate alone. With kubernetes-nmstate (and NodeNetworkState) however, we will always have the whole specification of network state in the object. It should be declarative, and keeping `state: absent` in the object seems wrong to me. What do you think?

Reply:
You could implement this logic and use the `absent` state at the backend only. You can even ask for a special verb on nmstate to do this.
However, please consider that this will cause a burden on kubernetes-nmstate to manage all the node, instead of managing only what interests the operator/admin.
It is reasonable for an admin to reserve interfaces or other resources to itself and not let k8s take ownership on them. Even if you will not need this today, and it will appear somewhere in the future, you will have hard time to support it.

Also, consider that you may want today to ignore specific interfaces all together (like tap, veth and alike), as they are not "owned" by the node, but by the pods.

Reply:
@EdDev thanks, tracking the issue in https://github.com/nmstate/kubernetes-nmstate/issues/48. I think it is worth a bigger discussion once we have working project and some real use cases. Thanks for your input